README.md

```diff
@@ -92,11 +92,17 @@ It is intended to reduce boiler-plate code around the API calls.
 In addition, the classes `RedactInstance` and `RedactJob` provide convenient high-level access to the API:
 
 ```python
-from redact import RedactInstance
+from redact import RedactInstance, ServiceType, OutputType
 
-redact = RedactInstance.create(service='blur', out_type='images', redact_url='http://127.0.0.1:8787')
+redact = RedactInstance.create(service=ServiceType.blur, output_type=OutputType.images, redact_url='http://127.0.0.1:8787')
```

Comment:
Maybe we can add an example of how to use it with our cloud? How to provide the api key?